packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -811,6 +813,12 @@ const getGridStyles = (
 
     border: 'none',
 
+    '.rdg-cell': {
+      '&:last-child': {
+        borderInlineEnd: 'none',
+      },
+    },
```

Comment:
before, this was on the `getCellStyles`, which isn't applied to non-"auto-like" cells anymore, so whether the last cell in a row had a right border wasn't deterministic.

Reply:
good catch :+1: